package.json

```diff
@@ -73,7 +73,7 @@
     "diff": "^5.0.0",
     "diff2html": "^3.1.18",
     "fhir": "^4.10.0",
-    "fhir-package-loader": "^0.3.0",
+    "fhir-package-loader": "^0.4.0",
```

Comment:
FPL is at v0.5.0 now, so... since you're bumping this anyway, might as well bump it to the latest.